examples/NewModuleTemplate/build.psake.ps1

```diff
@@ -120,6 +120,9 @@ Properties {
     $PublishRootDir = "$PSScriptRoot\.publish"
     $PublishDir     = "$PublishRootDir\$ModuleName"
 
+    # The local installation directory for the install task. Defaults to your user PSModulePath.
+    $InstallPath = "$($($env:PSModulePath).Split(';')[0])\$ModuleName"
+
```

Comment:
Are you sure the user's modules path will always be the first?  It might be better to just default to installing into `Join-Path (Split-Path $profile.CurrentUserAllHosts -Parent) "Modules\$ModuleName"`.  I've also experienced systems where $env:PSModulePath is fubar'd (no values).